src/workloads/execution/TimeSeriesLastpoint.yml

```diff
@@ -46,75 +61,191 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
 
-- Name: CreateIndex
+# Lastpoint query with ascending index on meta subfield and descending index on time.
```

Comment:
Another wording nit: this comment sounds like we have two separate indexes, one on meta and one on time. Maybe something like "compound index with meta subfield ascending and time descending"?